### ISSUE TITLE
Please add a license

### ISSUE THREAD

**Issue by levonk** [jexp/jpatterns#6]
Is this abandoned?   I hope not, thanks for your previous work.   If it is could you add a license so somebody else can pick it up?   Preferably Apache 2.0, MIT, BSD.

**Comment by Frankmartin** [comment]
It's Apache 2.0 see
http://www.jpatterns.org/license/

**Comment by kabutz** [comment]
Indeed, Apache 2.0.  A bit of history:  We wrote all the code really quickly back in 2010 and since it is primarily used for documentation, didn't see need for much in terms of changes.  But please feel free to contribute!  I use these a lot to teach my Design Patterns Course (http://www.javaspecialists.eu/courses/dpc.jsp and https://vimeo.com/ondemand/javadesignpatternscourse) and they are intended primarily for documentation purposes.  Some of my clients use them internally to document their patterns usage.
